libcodechecker/analyze/analyzers/result_handler_base.py

```diff
@@ -176,10 +176,11 @@ def analyzer_result_file(self):
             analyzed_file = self.analyzed_source_file
             _, analyzed_file_name = os.path.split(analyzed_file)
 
-            uid = str(uuid.uuid1()).split('-')[0]
+            file_name = str(self.buildaction.analyzer_type) + '_' + \
+                self.buildaction.original_command
 
-            out_file_name = str(self.buildaction.analyzer_type) + \
-                '_' + analyzed_file_name + '_' + uid + '.plist'
+            out_file_name = analyzed_file_name + '_' + \
+                hashlib.md5(file_name).hexdigest() + '.plist'
```

Comment:
I was puzzled for a bit as I specifically tested by hand if multiple files are analyzed in with different compilation setups in the buildlog and I properly got the plists duplicated with content appropriate for the build environment.

If you see right above in the previous line, the `file_name` variable is created from the original build command. This variable should be renamed, though, it seems to confuse people who don't read diff with enough context. :slightly_smiling_face: